pubspec.yaml

```diff
@@ -17,28 +17,30 @@ dependencies:
   flutter_lints: ^5.0.0
   get_it: ^8.0.2
   injectable: ^2.5.0
-  json_serializable: ^6.9.0
+  json_serializable: ^6.8.0
   json_annotation: ^4.9.0
-  build_runner: ^2.4.13
   auto_route: ^9.2.2
-  auto_route_generator: ^9.0.0
   cached_network_image: ^3.4.1
   dio: ^5.7.0
   freezed_annotation: ^3.0.0
   encrypted_shared_preferences: ^3.0.1
   flutter_bloc: ^8.1.6
-  mockito: ^5.4.6
+  mockito: ^5.4.4 # ✅ compatible with hive_generator
```

Comment:
`hive_generator` is incompatible with freezed + auto_route_generator libraries. I think we should move away of a strategy using hive a data base library